app/helpers/notifications_helper.rb

```diff
@@ -76,13 +74,13 @@ def notification_link(notification)
     else
       notifiable.details
```

Comment:
According to the above cited gist, possibly `"".html_safe()+notifiable.details` (and remove `text.html_safe`)?